### ISSUE TITLE
cms plugin magic table naming removal

### ISSUE THREAD

**Issue by FinalAngel** [aldryn/aldryn-disqus#3]
```
ProgrammingError at /en/admin/cms/page/clear-placeholder/1/
relation "aldryn_disqus_disqusplugin" does not exist
LINE 1: ...dryn_disqus_disqusplugin"."cmsplugin_ptr_id" FROM "aldryn_di...
```

**Comment by stefanfoulis** [comment]
This is due to aldryn-disqus not renaming its tables to not use the cms plugin table renaming magic anymore and is a problem with django CMS 3.1 and up.

**Comment by czpython** [comment]
Fixed by 4c06bfe7d5019c166b5e7ba432020201ae343746
